content/docs/command-reference/unfreeze.md

```diff
@@ -61,7 +61,7 @@ foo.dvc:
 ```
 
 DVC notices that `foo` changed due to the `foo.dvc` file that tracks this file
-(as `outs`), but the `make_copy` stage doesn't records the change among it's
+(as `outs`), but the `make_copy` stage no longer records the change among its
```

Comment:
"No longer" would mean it did before, which isn't shown.
```suggestion
(as `outs`), but the `make_copy` stage doesn't record the change among its
```

Reply:
My mistake, I thought this made it consistent with wording `freeze` but I see now the tense should be different.